<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>236</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>236</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>226</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#00007f;&quot;&gt;Проводим запрос погоды по аэропортам&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_UUWW">
      <property name="minimumSize">
       <size>
        <width>224</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>Запрос Внуково</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_UUDD">
      <property name="minimumSize">
       <size>
        <width>224</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Запрос Домодедово</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_UUEE">
      <property name="minimumSize">
       <size>
        <width>224</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Запрос Шереметьево</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
